### PR TITLE
tex-fmt: add page

### PR DESCRIPTION
**Pull request by WGUNDERWOOD**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page(s) follow the [style guide](/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):** tex-fmt v0.4.7

### CODE REVIEW

File: pages/common/tex-fmt.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+- Format a file, overwriting the original file in-place:
+
+`tex-fmt {{path/to/file.tex}}`
+
+- Check if a file is correctly formatted:
+
+`tex-fmt --check {{path/to/file.tex}}`
+
+- Format a file from stdin and print to stout:
```
Comment (spageektti) [inline pages/common/tex-fmt.md:14]:
```suggestion
- Format a file from `stdin` and print to `stout`:
```

**Review by spageektti — COMMENTED** [review]
Hi. Thank you so much for your contribution!

File: pages/common/tex-fmt.md (hunk 1 above)
Comment (WGUNDERWOOD) [reply pages/common/tex-fmt.md]: Thanks, I have fixed this

File: pages/common/tex-fmt.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+`cat {{/path/to/file.tex}} | tex-fmt --stdin`
```
[the remark this answers is not in the document]
Comment (WGUNDERWOOD) [reply pages/common/tex-fmt.md]: Thanks, I have fixed this

File: pages/common/tex-fmt.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
+# tex-fmt
+
+> Format LaTeX source code.
+> More information: <https://github.com/WGUNDERWOOD/tex-fmt>.
+
+- Format a file, overwriting the original file in-place:
```
Comment (gutjuri) [inline pages/common/tex-fmt.md:6]:
```suggestion
- Format a file, overwriting the original file:
```

Overwriting is always in-place :)

File: pages/common/tex-fmt.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+`tex-fmt {{path/to/file.tex}}`
+
+- Check if a file is correctly formatted:
+
+`tex-fmt --check {{path/to/file.tex}}`
+
+- Format a file from `stdin` and print to `stout`:
```
Comment (gutjuri) [inline pages/common/tex-fmt.md:14]:
```suggestion
- Format a file read from `stdin` and print to `stout`:
```

**Review by gutjuri — COMMENTED** [review]
Thanks, I have two suggestions :)

File: pages/common/tex-fmt.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+
+- Format a file, overwriting the original file:
+
+`tex-fmt {{path/to/file.tex}}`
+
+- Check if a file is correctly formatted:
+
+`tex-fmt --check {{path/to/file.tex}}`
+
+- Format a file read from `stdin` and print to `stout`:
```
Comment (gutjuri) [inline pages/common/tex-fmt.md:14]:
```suggestion
- Format a file read from `stdin` and print to `stdout`:
```

One more typo I previously overlooked... LGTM after that. Thanks for incorporating my previous suggestions!

**Review by gutjuri — APPROVED** [review]

File: pages/common/tex-fmt.md (hunk 5 above)
Comment (WGUNDERWOOD) [reply pages/common/tex-fmt.md]: Thanks, fixed

**Pull request merged**